lib/fetch/sanitize.spec.js

```diff
@@ -288,8 +288,115 @@ describe('data/sanitize', () => {
         github: 'http://github.com',
         twitter: 'http://twitter.com',
         categories: [
-          'Engineering',
+          { name: 'Engineering', slug: 'engineering' },
+          { name: 'Leadership', slug: 'leadership' },
+        ],
+      });
+    });
+
+    it('converts valid tags into categories', () => {
```

Comment:
@asavin Yes, remember we decided that when you sat with me to create the badger type in prismic. This PR doesn't add that, that was already there in the original [Badgers PR](https://github.com/redbadger/badger-brain/pull/87). This PR is just about returning the slug along with the category and stopping invalid categories from being returned